home.js

```diff
@@ -8,60 +8,86 @@ const { button } = require('./blocks/elements/button')
 const { DateTime } = require("luxon");
 
 const SHOW_DAYS_UNTIL = 10
+const DAYS_IN_WEEK = 5;
 const format = {...DateTime.DATETIME_MED, month: 'long' };
 
 /**
  * Updates the App-Home page.
  */
 const update = async (client, userId) => {
-  const today = DateTime.now()
-  const days = dfunc.listNWeekdays(today, SHOW_DAYS_UNTIL)
-  let blocks = []
-
-  blocks = blocks.concat(
-    plain_text(`Tiedot päivitetty ${today.setZone("Europe/Helsinki").setLocale('fi').toLocaleString(format)}`),
-    actions([
-      button('Päivitä', 'update_click', 'updated')
-    ]),
-    divider()
-  )
-
-  for (let i = 0; i < days.length; i++) {
-    const d = days[i]
-
-    blocks = blocks.concat(
-      header(dfunc.toPrettyFormat(d))
-    )
-    const enrollments = await service.getEnrollmentsFor(d)
-    let usersString = enrollments.length === 0 ? "Kukaan ei ole ilmoittautunut toimistolle!" : "Toimistolla aikoo olla:\n"
-    enrollments.forEach((user) => {
-      usersString += `<@${user}>\n`
+    let settings = await getSettingsPage(userId)
+    let updateBlock = await getUpdateBlock()
+    let enrollments = await getEnrollmentsPage(userId)
+    let blocks = settings.concat(updateBlock, enrollments)
```

Comment:
```suggestion
    let blocks = []
    blocks = blocks.concat(
        await getSettingsBlock(userId), 
        await getUpdateBlock(), 
        await getEnrollmentsBlock(userId)
    )
```
lisäksi kyseisten funktioiden ja niiden muuttujien uudelleennimentä vastaaviksi

Reply:
Joo, tuo on selkeämpi, muutetaan :)